packages/plexus/package.json

```diff
@@ -24,23 +24,24 @@
     "@types/d3-zoom": "3.0.3",
     "@types/react": "16.8.7",
     "@types/react-dom": "18.2.5",
+    "@typescript-eslint/eslint-plugin": "^6.4.0",
+    "@typescript-eslint/parser": "^6.4.0",
     "babel-loader": "9.1.3",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "clean-webpack-plugin": "4.0.0",
     "css-loader": "6.8.1",
-    "enzyme": "3.11.0",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
one thing I don't understand with our setup - what is the relationship between dependencies in the top-level package.json and subpackage-level? If something is at top level, is it just inherited by sub-packages?